benchmarks/torchbench/benchtest.yaml

```diff
@@ -24,6 +24,10 @@ defaults: &defaults
       - descr
       - progress
       - total
+      - gpu_time
+      - cpu_dispatch_time
+      - cpu_walltime
+      - tflops
```

Comment:
That is indeed what you need to do, so I am not sure why it doesn't work.

Reply:
It actually works if the `#endloop__ as step` lines are removed. Is it because the bench is flagged as finished (at the end of the loop) before it can register the custom gives? In that case, does that mean the only the first time the loop is executed is currently registered? There could be a problem with torchbench then as there is a warmup phase before the main loop:

https://github.com/pytorch/benchmark/blob/ff7114655294aa3ba57127a260dbd1ef5190f610/run.py#L55-L85

Reply:
The `--stop` flag does a form of early stopping. We could turn it off for torchbench since it's already designed to only run a limited number of steps.